docs/technical-reference.md

```diff
@@ -208,7 +208,30 @@ Note the comma after the object. Your trust relationship should now look somethi
 }
 ```
 
-You can now configure a Quilt Role with this role (using the Catalog's admin panel, or `quilt3.admin.create_role`).
+You can now configure a Quilt Role with this role (using the Catalog's Admin panel, or `quilt3.admin.create_role`).
+
+### ManagedUserRoleExtraPolicies
+
+The `ManagedUserRoleExtraPolicies` parameter allows you to add additional IAM
+policies to the managed user role. This is useful for granting additional
+permissions to users in your Quilt instance, which otherwise would be blocked by
+the permission boundary. 
+
+This parameter works in conjunction with Quilt role configuration in the Admin panel.
+You need to:
+1. Create the appropriate IAM policy
+2. Add it to `ManagedUserRoleExtraPolicies` (this step)
+3. Create a Quilt role in the Admin panel that uses the managed user role (Source=Quilt)
```

Comment:
I think it should be something like this:

3. create Quilt policy with "Manually set ARN instead of configuring per-bucket permissions" flag and enter policy ARN
4. attach that Quilt policy to managed Quilt roles